docker/Dockerfile-workers

```diff
@@ -22,4 +24,6 @@ RUN dnf -y install \
 COPY . .
 RUN pip3 install . --no-deps
 EXPOSE 8080
-CMD ["/bin/celery-3", "-A", "cachito.workers.tasks", "worker", "--loglevel=info"]
+CMD ["export USER_ID=$(id -u)", "export GROUP_ID=$(id -g)", "envsubst < ${HOME}/passwd.template > /tmp/passwd", \
+    "export LD_PRELOAD=/usr/lib64/libnss_wrapper.so", "export NSS_WRAPPER_PASSWD=/tmp/passwd", \
+    "export NSS_WRAPPER_GROUP=/etc/group", "/bin/celery-3", "-A", "cachito.workers.tasks", "worker", "--loglevel=info"]
```

Comment:
There's no newline break here, could you add one?

Reply:
Also, should this become an "entrypoint.sh" script?

Reply:
Any specific benefits for the same? IMO, since this is temp. and we might be removing it soon once the upstream patch is submitted, there might not be a need for it. wdyt?

Reply:
The benefit of entrypoint is that it would probably reduce the size of this command, but the downside is that OpenShift's handling of entrypoint and command is confusing: https://docs.openshift.com/container-platform/3.11/dev_guide/deployments/basic_deployment_operations.html#executing-commands-inside-a-container-deployments

I don't have a strong preference either way.

Reply:
The definition for `CMD` [1] in the current format is that all other entries after the first one sould be considered parameters for the command being run.

In that list, I see 2 different commands being executed (envsubst and /bin/celery-3).

Moreover, I see no reason for getting rid of this approach in the future, since it may save us from future issues regarding some library consulting /etc/passwd.

[1] https://docs.docker.com/engine/reference/builder/#cmd

Reply:
TBH, I do not fully understand why whatever follows `envsubst` is not being processed as parameters for that command.

@lcarva, any ideas?

Reply:
Hmm good point @athos-ribeiro The doc mentions _There can only be one CMD instruction in a Dockerfile. If you list more than one CMD then only the last CMD will take effect._

Doesn't it mean that it's ignoring everything else that we added and only running the last one? :thinking:

Reply:
I got rid of all the additional commands that we added and the git-submodule support still works fine :brain: